src/wizard/wizard.scss

```diff
@@ -4,47 +4,46 @@
 
 @mixin iui-wizards {
   @include iui-reset;
-
   @include iui-font-family;
 
   > .iui-wizards-wrapper {
     display: flex;
     flex-direction: row;
     justify-content: space-around;
+    align-items: flex-start;
     width: 100%;
     margin-bottom: $iui-baseline / 2;
 
-    > .iui-wizards-step {
+    .iui-wizards-step {
       align-items: center;
-
-      @include themed {
-        color: t(iui-color-background-positive);
-      }
       display: flex;
       flex-direction: column-reverse;
       font-size: $iui-font-size-small;
       font-weight: $iui-font-weight-normal;
       line-height: $iui-line-height;
       position: relative;
+      @include themed {
+        color: t(iui-color-background-positive);
+      }
 
       &:hover {
         text-decoration: none;
       }
 
-      > .iui-wizards-step-title {
+      .iui-wizards-step-title {
+        text-align: center;
+        width: $iui-3xl * 2;
         @include themed {
           color: t(iui-text-color-muted);
         }
+
+        @media screen and (max-width: 1024px) {
+          width: $iui-xxl * 2;
+        }
```

Comment:
So this is all the PR is supposed to change after all? 😄 

Not sure how you arrive at these values but it's alright I guess.

Reply:
Replaced this with `width: 20vW` instead.

Reply:
probably should be lowercase `vw` for consistency

Reply:
setting `width: 20vw` is fine as a default, but I think we'll have to override it in react if there are more than 5 items